modules/bookList.js

```diff
@@ -0,0 +1,60 @@
+// Book Class: Represents a Book
+
```

Comment:
- [ ] According to the project requirements, you should omit the usage of the function keyword to declare functions across the code base. instead please use the arrow function declaration.

Reply:
Thanks Ill do that